### DISCUSSION TITLE
can a JSXExpressionContainer be wrapped with another?

### DISCUSSION THREAD

**Discussion by BoyaZhang** [babel/babel#11642] [Q&A] [answered]
i want to traverse a JSXExpressionContainer, something like {this.state.name} to {{this.state.name}} How does this work？..

**Comment by nicolo-ribaudo** [comment, marked as answer]
No, a JSXExpressionContainer can only contain a valid JS expression and not another JSXExpressionContainer. What is your use case?

**Comment by giuliohome** [comment]
Maybe is [this an example](https://github.com/richardfuca/react-native-decompiler/issues/243#issuecomment-1013154663)?
